Resources/Prototypes/DeltaV/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -0,0 +1,278 @@
+- type: entity
+  suffix: Serenitol 10u
+  parent: Pill
+  id: PillSerenitol
+  components:
```

Comment:
why is every prototype in this file in a different fucking order

Reply:
lmao, copy pasted upstream pill and pill canister without looking and just found out they aren't following their own docs. Will fix.